### DISCUSSION TITLE
How to set the maximum number of editable lines in Theia code editor?

### DISCUSSION THREAD

**Discussion by fjxfirst** [microsoft/monaco-editor#4295] [Q&A]
I want to set the maximum number of lines to 9999 in Theia. When pressing Enter or pasting, if it exceeds 9999 lines, I want Enter and paste to be disabled, and a message prompt ‘Exceeded the maximum of 9999 lines’ to be displayed

**Comment by hediet** [comment]
Please reach out to theia support forums for this.
